lib/std/os/linux/bpf/btf_ext.zig

```diff
@@ -0,0 +1,19 @@
+pub const Header = packed struct {
+    magic: u16,
+    version: u8,
+    flags: u8,
+    hdr_len: u32,
+
+    /// All offsets are in bytes relative to the end of this header
+    func_info_off: u32,
+    func_info_len: u32,
+    line_info_off: u32,
+    line_info_len: u32,
+};
+
+pub const InfoSec = packed struct {
+    sec_name_off: u32,
+    num_info: u32,
+    // TODO: communicate that there is data here
```

Comment:
Possible usecase for #4016

Reply:
Yeah that would be an excellent addition to Zig, we get to put that wasted billion dollars to use and get to emulate something available in C.